spec/models/pawn_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe Pawn, type: :model do
+  describe '#create' do
+    it 'creates a pawn in the database' do
+      pawn = FactoryGirl.create(:pawn)
+      expect(pawn.type == 'Pawn').to eq(true)
+    end
+  end
+
+  describe '#valid_pawn_move?' do
+    it 'returns true if the white pawn successfully has a normal move' do
```

Comment:
Can this description be a little more specific? What does "a normal move" mean in this context?